src/libtiled/logginginterface.h

```diff
@@ -46,6 +46,8 @@ class LoggingInterface
       INFO, ERROR
     };
 
+    virtual ~LoggingInterface() {}
```

Comment:
Nobody ever calls `delete` on a `LoggingInterface` pointer, and any class implementing this interface is also inheriting from `QObject` which already has a virtual destructor. But anyway I think this addition is fine.